rcon/player_history.py

```diff
@@ -13,6 +14,9 @@
 from rcon.game_logs import on_connected, on_disconnected
 from rcon.commands import CommandFailedError
 
+from rcon.steam_utils import get_player_bans, STEAM_KEY
+AUTOBAN_TRIGGER_THRESHOLD = os.getenv('BAN_ON_VAC_HISTORY')
```

Comment:
It's also a possibilty to default to 0 here

Reply:
Also it'd be good to change the name of that variable a bit so that it contains the unit, DAYS in that case